lib/fortnox/api/types/enums.rb

```diff
@@ -21,6 +21,12 @@ def self.default
         'COOKING','SNOWPLOWING','GARDENING','BABYSITTING','OTHERCARE',
         'TUTORING','OTHERCOSTS'
       )
+      CurrentHouseWorkTypes = Types::Strict::String.enum(
+        'CONSTRUCTION','ELECTRICITY','GLASSMETALWORK','GROUNDDRAINAGEWORK',
+        'MASONRY','PAINTINGWALLPAPERING','HVAC','CLEANING','TEXTILECLOTHING',
+        'SNOWPLOWING','GARDENING','BABYSITTING','OTHERCARE', 'OTHERCOSTS'
+      )
+      LegacyHouseWorkTypes = Types::Strict::String.enum( 'COOKING', 'TUTORING' )
```

Comment:
I tried that, but I get errors like
```
Dry::Types::ConstraintError:
       "[\"CONSTRUCTION\", \"ELECTRICITY\", \"GLASSMETALWORK\", \"GROUNDDRAINAGEWORK\", \"MASONRY\", \"PAINTINGWALLPAPERING\", \"HVAC\", \"CLEANING\", \"TEXTILECLOTHING\", \"SNOWPLOWING\", \"GARDENING\", \"BABYSITTING\", \"OTHERCARE\", \"OTHERCOSTS\", \"COOKING\", \"TUTORING\"]" violates constraints (included_in?([["CONSTRUCTION", "ELECTRICITY", "GLASSMETALWORK", "GROUNDDRAINAGEWORK", "MASONRY", "PAINTINGWALLPAPERING", "HVAC", "CLEANING", "TEXTILECLOTHING", "SNOWPLOWING", "GARDENING", "BABYSITTING", "OTHERCARE", "OTHERCOSTS", "COOKING", "TUTORING"]], "[\"CONSTRUCTION\", \"ELECTRICITY\", \"GLASSMETALWORK\", \"GROUNDDRAINAGEWORK\", \"MASONRY\", \"PAINTINGWALLPAPERING\", \"HVAC\", \"CLEANING\", \"TEXTILECLOTHING\", \"SNOWPLOWING\", \"GARDENING\", \"BABYSITTING\", \"OTHERCARE\", \"OTHERCOSTS\", \"COOKING\", \"TUTORING\"]") failed)
```

and
```
       expected Dry::Types::ConstraintError, got #<NoMethodError: undefined method `downcase' for #<Array:0x00560990bee4a8>> with backtrace:
         # ./spec/fortnox/api/types/examples/enum.rb:33:in `block (5 levels) in <top (required)>'
         # ./spec/fortnox/api/types/examples/enum.rb:38:in `block (6 levels) in <top (required)>'
         # ./spec/fortnox/api/types/examples/enum.rb:39:in `block (5 levels) in <top (required)>'
```

It appears as if the enum constructor does not accept an array as argument.